config/containers/runmetrics.md

```diff
@@ -78,10 +125,16 @@ in `docker ps`, its long ID might be something like
 look it up with `docker inspect` or `docker ps --no-trunc`.
 
 Putting everything together to look at the memory metrics for a Docker
-container, take a look at `/sys/fs/cgroup/memory/docker/<longid>/`.
+container, take a look at `/sys/fs/cgroup/memory/docker/<longid>/` on cgroup
```

Comment:
Is this true with the default systemd cgroups?

Reply:
updated